corehq/apps/smsbillables/static/smsbillables/js/smsbillables.rate_calc.js

```diff
@@ -10,144 +10,145 @@ hqDefine("smsbillables/js/smsbillables.rate_calc", [
     _,
     select2Handler
 ) {
-    var SMSRateCalculator = function (formData) {
-        'use strict';
-        var self = {};
-
-        self.gateway = ko.observable();
-        self.direction = ko.observable();
-        self.select2CountryCode = select2SmsRateHandler(formData.country_code);
-        self.rate = ko.observable();
-        self.hasError = ko.observable(false);
-        self.noError = ko.computed(function () {
-            return ! self.hasError();
-        });
-        self.calculatingRate = ko.observable(false);
-        self.showRateInfo = ko.computed(function () {
-            return self.rate() && ! self.calculatingRate();
-        });
+    $(function () {
```

Comment:
Same question as above on moving model definitions into the document ready handler.